test/list-and-get.test.js

```diff
@@ -0,0 +1,141 @@
+// SPDX-FileCopyrightText: 2022 Mix Irving
+//
+// SPDX-License-Identifier: LGPL-3.0-only
+
+const test = require('tape')
+const { promisify: p } = require('util')
+const pull = require('pull-stream')
+const ref = require('ssb-ref')
+const Testbot = require('./helpers/testbot')
+
+test('tribes.list + tribes.get', (t) => {
+  const name = `list-and-get-groups-${Date.now()}`
+  let server = Testbot({ name })
+  const keys = server.keys
+
+  server.tribes2.create(null, (err, group) => {
+    t.error(err, 'create group')
+
+    pull(
+      server.tribes2.list(),
+      pull.collect(async (err, list) => {
+        if (err) return t.error(err)
+
+        const expectedGroup = {
+          secret: group.secret,
+          root: group.root,
+          id: group.id,
+        }
+
+        t.deepEqual(list, [expectedGroup], 'lists group ids')
+
+        server.tribes2.get(group.id, (err, actualGroup) => {
+          if (err) throw err
+
+          t.deepEqual(actualGroup, expectedGroup, 'gets group data')
+
+          server.close((err) => {
+            t.error(err, 'closes server')
+
+            server = Testbot({ name, rimraf: false, keys })
+            pull(
+              server.tribes2.list(),
+              pull.collect((err, newList) => {
+                if (err) throw err
+
+                t.deepEqual(
+                  newList,
+                  list,
+                  'list returns save results after restart'
+                )
+                server.close(true, t.end)
+              })
+            )
+          })
+        })
+      })
+    )
+  })
+})
+
+test.skip('tribes.list (subtribes)', async (t) => {
```

Comment:
We should not add a test that is already skipped. Just delete it

Reply:
i was thinking we would have an easier time later on if we import whole test files and not parts of them, makes for less figuring out once we actually add e.g. subtribes

Reply:
"import" you mean from tribes1 to tribes2?

Reply:
Yes. In the interest of not lowering test coverage.

Reply:
Are there more tests to still be imported? Or are they all now in this PR?

If there are still more tests to be imported, then this one here could be added next time we import more tests.

Reply:
There are a lot of tests to be imported about subtribes and poboxes etc. But they're all in other files in tribes1. So I was thinking a single test like this might be easy to miss.

Reply:
Okay. Let's resolve this, but let's aim at making gh issues, not code TODOs as issues.